zebra-script/Cargo.toml

```
@@ -23,7 +23,9 @@ comparison-interpreter = []
 
 [dependencies]
 zcash_script = { workspace = true }
-zebra-chain = { path = "../zebra-chain", version = "1.0.0" }
+zcash_primitives = { workspace = true }
+
+zebra-chain = { path = "../zebra-chain", version = "1.0.0-beta.46" }
```

```suggestion
zcash_primitives = { workspace = true }
zebra-chain = { path = "../zebra-chain", version = "1.0.0" }
```

(or `workspace = true` for it too?)

---

> (or `workspace = true` for it too?)

That doesn't work.

---

Commited in [0bdd5b2](https://github.com/ZcashFoundation/zebra/pull/9670/commits/0bdd5b28e90458f1f4f6991d3ce4672b16fb5917).